cluster/gce/configure-vm.sh

```diff
@@ -851,6 +851,17 @@ function node-docker-opts() {
   if [[ -n "${EXTRA_DOCKER_OPTS-}" ]]; then
     DOCKER_OPTS="${DOCKER_OPTS:-} ${EXTRA_DOCKER_OPTS}"
   fi
+
+  # Decide whether to enable the cache. Both of these variables
+  # come from the "kube-env" metadata value.
+  if [[ "${ENABLE_DOCKER_REGISTRY_CACHE:-}" == "true" && -n "${ZONE:-}" ]]; then
+    REGION=$(echo "${ZONE}" | cut -f 1 -d -)
+    if [[ "${REGION}" == "europe" ]]; then
+      REGION="eu"
+    fi
```

Comment:
GitHub seems to have eaten my comment in the UI, so apologies if this is a repeat. I'd rather see this moved up to util.sh and the variable be something like `DOCKER_REGISTRY_CACHE_REGION`, since we already have this hacky logic encapsulated here: https://github.com/kubernetes/kubernetes/blob/782de9d02417401bc53fb0eee6c79f7b3b9ee911/cluster/gce/util.sh#L169